### ISSUE TITLE
[BUG] Seek and Volume Bar not responsive enough

### ISSUE THREAD

**Issue by Drikani** [LeoKlaus/plappa#240]
**Describe the bug**
Touch target for seeking and volume bar is too small and more often than not it will not trigger but try to minimize the player

**To Reproduce**
1. Play Audiobook
2. Try to seek to new play position or change volume

**Expected behavior**
The touch target should be larger and it should not be possible to minimize the player when touching in the area of the seeking or volume bar

**Environment(please complete the following information):**
 - Device: iPhone 15 Pro
 - OS: iOS 18.2
 - Version 1.5.0 (78)
 - Server Version ABS 2.17.5

**Comment by Drikani** [comment]
I don't know if this is a bug or more a feature request?
I would really like it if it would work more like within Apple Music as there it never happens that I need to focus when seeking or changing volume.
Within Plappa it always is a little bit finnicky

**Comment by cwhite616** [comment]
I feel exactly the opposite… I have an iPhone 16 with an always on screen, and the number of times I totally screw up my book because casual handing of my phone completely moves the seek bar… Well, it makes me quite a bit unhappy. I have open settings a number of times Looking for a way to totally disable the seek bar, because I cannot remember a time where I have ever found it useful, and in my experience it does nothing but screw up my listening.

If the seek bar was made “more responsive“ I would probably start to look for another playback app. If we’re looking at future requests related to the Ck bar, I would truly love an option to disable it altogether.

**Comment by fallmake** [comment]
I understand this issue is about seek bar in the app, not on the lock screen. See issues #295 and #354 for the lock screen bar.

In the open app while playing a book, I can hit the seek bar and move it somewhat consistently, but after an initial seek, it feels unresponsive for some seconds (debounce?). That prevents me from seeking to the exact time I want by adjusting it a bit ahead or back. At that point it’s as in the initial description here. I don’t have the same issue with the volume bar. (Iphone 15 pro max, version 26.2, plappa 1.5.6)

**Comment by masonfox** [comment]
I agree, I love Plappa, but the lack of consistent responsiveness with the seekbar can be frustrating!

Please fix this! 🙏
